### DISCUSSION TITLE
Admin panel in mobile has overflow elements

### DISCUSSION THREAD

**Discussion by manuelserol** [bagisto/bagisto#10356] [General]
When logged in as admin in mobile, the navbar and dashboard elements are overflowing

Bagisto v2.2.1
Browser - Any

**Comment by amit-webkul** [comment]
Hello @manuelserol 

Thank you for bringing this to our attention regarding the navigation bar and dashboard layout when accessing the admin panel on mobile devices.

We would like to clarify that the Bagisto admin panel is currently optimized for desktop and tablet devices, as the majority of users access the admin interface on these platforms for efficient management and data entry tasks. While mobile responsiveness is not a primary focus for the admin panel, we continuously monitor user needs and feedback to prioritize future improvements.

Let us know your thoughts or if there are other concerns we can assist you with.

Best regards,
